### ISSUE TITLE
Map result format locations parameter breaks when one value is invalid 

### ISSUE THREAD

**Issue by JeroenDeDauw** [JeroenDeDauw/Maps#512]
Seen with Maps 5.6

Demo: https://www.transit.wiki/Sandbox_3

```
Without trailing ;

<div class=bigmap>{{#ask: [[-Place::14-Mission]]|?latlong|format=googlemaps|width=100%|height=100%|icon=Mm_20_red.png
|locations=37.744693, -122.426165~D~D~Bus20.png;37.768580, -122.501696~V~V~Bus20.png
}}</div>


With trailing ;
<div class=bigmap>{{#ask: [[-Place::14-Mission]]|?latlong|format=googlemaps|width=100%|height=100%|icon=Mm_20_red.png
|locations=37.744693, -122.426165~D~D~Bus20.png;37.768580, -122.501696~V~V~Bus20.png;
}}</div>
```

**Comment by JeroenDeDauw** [comment]
@acnetj I'd be cool if you could switch from 5.6 to the latest 6.x as a lot of changes have been made since and some issues you are running into might already have been fixed. The latest release certainly has less bugs than 5.6.

**Comment by acnetj** [comment]
To do that I will have to upgrade MW from 1.27 to 1.31, which will require some preparation, but everything else like PHP has been upgraded.

**Comment by JeroenDeDauw** [comment]
Fixed by https://github.com/JeroenDeDauw/ParamProcessor/commit/84cc78d8aa3a979d6f67360c5bdde75a2c0d1757

**Comment by JeroenDeDauw** [comment]
Fix will be in 6.3

**Comment by JeroenDeDauw** [comment]
Now released as 6.2.1
